website/styles/index.css

```diff
@@ -142,6 +142,95 @@ hr {
 	color: var(--accent-color);
 }
 
+/* Slideshow container */
+.slideshow-container {
+	max-width: 100%;
+	max-height: 250px;
+	position: relative;
+	margin: auto;
+	padding: 20px;
+	min-width: 100%;
+
+  }
+
+.slide {
+	max-width:100%;
+	max-height:180px;
+	height: auto;
+	object-fit: scale-down;
+	margin: auto;
+}
+
+/* Next & previous buttons */
+.arrow {
+	cursor: pointer;
+	position: absolute;
+	top: 50%;
+	width: auto;
+	padding: 16px;
+	margin-top: -22px;
+	color: var(--secondary-text-color);;
+	font-weight: bold;
+	font-size: 18px;
+	transition: 0.6s ease;
+	}
```

Comment:
Needs a space

Reply:
Everything fixed, also repositioned the pictures so that they were cropped in places where it didn't affect the picture by cutting out something important.